core/tests/test_utils.py

```diff
@@ -231,15 +289,45 @@ def _parse_json_response(self, json_response, expect_errors=False):
         return json.loads(json_response.body[len(feconf.XSSI_PREFIX):])
 
     def get_json(self, url, params=None, expect_errors=False):
-        """Get a JSON response, transformed to a Python object."""
+        """Get a JSON response, transformed to a Python object.
+
+        Args:
+            url: string (url). Url for the api get request.
+            params: dict (json). default (None).
+                parameters relevant to the api call.
+            expect_errors: bool. If False, then if anything is written
+                to wsgi.errors (module webtest) will be an error.
+
+        Returns:
+            json (object). Response from get api call is returned
+                as json object.
+        """
         json_response = self.testapp.get(
             url, params, expect_errors=expect_errors)
         return self._parse_json_response(
             json_response, expect_errors=expect_errors)
 
     def post_json(self, url, payload, csrf_token=None, expect_errors=False,
                   expected_status_int=200, upload_files=None):
-        """Post an object to the server by JSON; return the received object."""
+        """Post an object to the server by JSON; return the received object.
+
+        Args:
+            url: string (url). url for the api post request.
```

Comment:
Here and elsewhere, start the part after the type with a capital letter. E.g. "url: str. The URL to make a POST request to."

Reply:
done

Reply:
done

Reply:
done